hawkbit-mgmt-resource/src/test/java/org/eclipse/hawkbit/mgmt/rest/resource/MgmtSoftwareModuleTypeResourceTest.java

```diff
@@ -337,7 +338,7 @@ public void invalidRequestsOnSoftwaremoduleTypesResource() throws Exception {
                 .contentType(MediaType.APPLICATION_JSON)).andDo(MockMvcResultPrinter.print())
                 .andExpect(status().isBadRequest());
 
-        final SoftwareModuleType missingName = entityFactory.generateSoftwareModuleType("test123", null, "Desc123", 5);
+        final SoftwareModuleType missingName = new JpaSoftwareModuleType("test123", null, "Desc123", 5);
```

Comment:
The entity factory ensures that no illegal entities are created. However, in this case the whole idea is to create an illegal one.